modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/1_application.json

```
@@ -55,16 +55,16 @@
 			"productName": "Home"
 		},
 		{
-			"applicationCreateDate": "2022-09-09",
+			"applicationCreateDate": "2022-09-09T13:00:00Z",
 			"applicationStatus": "quoted",
 			"email": "joana@liferay.com",
 			"externalReferenceCode": "RAYAPP-007",
-			"firstName": "Joana",
+			"firstName": "Eduardo",
```

Why do you change it?
```suggestion
			"firstName": "Joana",
```

---

Just for testing the sort date functionality.